lib/rules/landmark-unique-matches.js

```diff
@@ -0,0 +1,39 @@
+const excludedParentsForHeaderFooterLandmarks = [
```

Comment:
Based on testing with VO I think this filtering is unnecessary but need more data

Reply:
for the record: we have decided that because this is a best-practice rule, it will follow the [ARIA 1.1 practices](https://www.w3.org/TR/wai-aria-practices-1.1/#aria_landmark) document regardless of treatment by browser/AT combinations